### PR TITLE
fix(ui): adjust spacing around deploy options

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Fix the spacing around the deploy options.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- Visit the machine list, check a deployable machine and open the deploy form via the take action menu.
- Check that there is a nice amount of space between the two checkbox options.
- Reduce the size of your window until the labels wrap onto a new line. The wrapping text should be indented to match the label (and clear the checkbox).
- Now visit a deployable machine's details page and open the deploy form via the take action menu.
- Check that there is a nice amount of space between the two checkbox options.
- Reduce the size of your window until the labels wrap onto a new line. The wrapping text should be indented to match the label (and clear the checkbox).

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2040.

### CODE REVIEW

**Review by Caleb-Ellis — CHANGES_REQUESTED** [review]
This doesn't seem right:




Do the checkboxes need to be inline blocks?

**Comment by huwshimi** [comment]
@Caleb-Ellis Whoops! Fixed.

File: legacy/src/app/partials/nodedetails/deploy-options.html [hunk 1]
```diff
@@ -4,78 +4,65 @@
...
-            </p>
-          </div>
-        </div>
-      </li>
-      <li class="p-inline-list__item">
-        <a
-          class="p-link--external"
-          href="https://maas.io/docs/kvm-introduction"
-          target="_blank"
+        <p
```
Comment (Caleb-Ellis) [inline legacy/src/app/partials/nodedetails/deploy-options.html:32]: Do you think we can remove this `p` block? It's not in the react version and it seems redundant to show a warning when the checkbox warns the same thing.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM! Just one comment for consideration.

File: legacy/src/app/partials/nodedetails/deploy-options.html (hunk 1 above)
Comment (huwshimi) [reply legacy/src/app/partials/nodedetails/deploy-options.html:32]: I think you're right, pretty sure it was removed from the react version at some point, but not removed here.

**Pull request merged**